components/Form.tsx

```diff
@@ -86,15 +86,15 @@ export const Form = () => {
               />
             ))}
             <Button
-              type="button"
+              type="submit"
               onClick={() => {
                 setParticipants([...participants, { ...emptyParticipant }]);
               }}
             >
               {"Ajouter un participant"}
             </Button>
             <Button
-              type="submit"
+              type="button"
```

Comment:
Need to keep this one has submit, else we can't send the form

Reply:
Corrected